CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 # Changelog
+# 1.85.0
+* Enhancement: Remove unused "generic metrics" code and processing. [#4225](https://github.com/aws/amazon-ecs-agent/pull/4225)
```

Comment:
nit(non-blocking): typically we would use `-` instead of `:`

Reply:
Updated!

Reply:
thanks!